Libs/DeathNotificationLib/DeathNotificationLib.lua

```diff
@@ -1113,6 +1185,17 @@ local function handleEvent(self, event, ...)
 		C_Timer.After(5.0, function()
 			deathlogJoinChannel()
 		end)
+	elseif event == "DUEL_TO_THE_DEATH_REQUESTED" then
```

Comment:
Is there a reason you wanted to send a message after this event triggered? Could we just get the player names here for both sides?

Reply:
The message is needed, as the WoW API only informs the player, who gets challenged to a duel, that he is getting challenged, but not the person who is challenging. 

Therefore, once a person is getting challenged, he will send a message, to inform the other player about the duel also, so that in case the person who made the duel request is dying, he will correctly report the death as a duel death.